src/host.ts

```diff
@@ -2,9 +2,13 @@ import { Builder } from '@vtex/api'
 import * as retry from 'async-retry'
 import { map, reduce } from 'ramda'
 
+import * as moment from 'moment'
+import { getStickyHost, hasStickyHost, saveStickyHost } from './conf'
 import { BuilderHubTimeoutError } from './errors'
 import log from './logger'
 
+const TTL_SAVED_HOST_HOURS = 6
```

Comment:
Shouldn't this be 4?

Reply:
I'm not sure which number to pick here. Is there any reason why 4 would be better/more suitable than 6?

Reply:
I just thought we agreed on 4 before. But we can leave it at 6.